packages/terra-framework-site/src/Index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Router, Route, hashHistory, IndexRoute, Redirect } from 'react-router';
+import App from './App';
+import Home from './Home';
+
+// Examples
+// import AlertExamples from './examples/alert/Index';
+
+// Test Routes
+/* eslint-disable import/first */
+// import AlertTestRoutes from 'terra-alert/tests/nightwatch/AlertTestRoutes';
+import TestLinks from './TestLinks';
+
+ReactDOM.render((
+  <Router history={hashHistory}>
+    <Redirect from="/" to="/site" />
+    <Route path="/site" component={App}>
+      <IndexRoute component={Home} />
+      {/* <Route path="alert" component={AlertExamples} /> */}
```

Comment:
do you want to remove these commented out pieces here

Reply:
I'm leaving them here because I'm going to be immediately adding a new package after this gets merged and I want to make my life easier with some examples.